Libraries/Components/WebView/WebView.android.js

```diff
@@ -195,6 +198,15 @@ var WebView = React.createClass({
       console.warn('WebView: `source.body` is not supported when using GET.');
     }
 
+    var onShouldOverrideUrlLoading = this.props.onShouldStartLoadWithRequest
+        && ((event: Event) => {
+            var shouldOverride = this.props.onShouldStartLoadWithRequest(event.nativeEvent);
```

Comment:
@pglotov Possible issue based on my reading of the source here. The semantics of `onShouldStartLoadWithRequest` are that if a `true` value is returned, the WebView should handle the request, whereas the semantics of `shouldOverrideUrlLoading` are the same if a `false` value is returned. It seems to me that this line should be changed to:

```javascript
var shouldOverride = !this.props.onShouldStartLoadWithRequest(event.nativeEvent);
```